### ISSUE TITLE
Can't build main?  Do a complete clean - also Can't build main in VS 2019

### ISSUE THREAD

**Issue by dsyme** [dotnet/fsharp#11986]
Recent updates seem to now require a complete flush of the repo to build

```
git clean -xfd . 
.\build
```

Without this I got about 100 of these

```
  16 of 69 projects are up-to-date for restore.
C:\GitHub\dsyme\fsharp\artifacts\obj\FSharp.ProjectSystem.Base\Debug\net472\.NE
TFramework,Version=v4.7.2.AssemblyAttributes.cs(4,12): error CS0579: Duplicate
'global::System.Runtime.Versioning.TargetFrameworkAttribute' attribute [C:\GitH
ub\dsyme\fsharp\vsintegration\src\FSharp.ProjectSystem.Base\FSharp.ProjectSyste
m.Base.csproj]
```

**Comment by dsyme** [comment]
Sigh I'm still getting problems in `main`

When building inside Visual Studio 2019:

```

16>------ Build started: Project: LibraryProject, Configuration: Debug Any CPU ------
4>C:\Program Files\dotnet\sdk\5.0.303\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.targets(127,5): error NETSDK1141: Unable to resolve the .NET SDK version as specified in the global.json located at C:\GitHub\dsyme\fsharp\global.json.
4>Done building project "FSharp.UIResources.csproj" -- FAILED.
17>------ Build started: Project: TutorialProject, Configuration: Debug Any CPU ------
14>C:\Program Files\dotnet\sdk\5.0.303\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.targets(127,5): error NETSDK1141: Unable to resolve the .NET SDK version as specified in the global.json located at C:\GitHub\dsyme\fsharp\global.json.
14>Done building project "TaskPerfCSharp.csproj" -- FAILED.
```

**Comment by dsyme** [comment]
@vzarytovskii @KevinRansom Am I doing something wrong here?

**Comment by dsyme** [comment]
Looks like this https://github.com/dotnet/fsharp/pull/11963#issuecomment-900693051?

**Comment by dsyme** [comment]
If you can't build main in VS2019 and need a workaround @vzarytovskii  suggests setting the SDK and tools versions in global.json to

```
6.0.100-preview.6.21355.2
```

**Comment by vzarytovskii** [comment]
Uh, so, arcade updated sdk to the rc1 version, which isn't available yet in public feed, and it builds with build script but not from VS (for example when running tests), since build script installs sdk into .dotnet, and VS doesn't see it.

**Comment by vzarytovskii** [comment]
Roslyn has somewhat similar problem:

https://github.com/dotnet/roslyn/pull/55681

I will downgrade required sdk to preview and add some CI steps to make sure we always use publicly available one.

**Comment by JoeRobich** [comment]
> Roslyn has somewhat similar problem:

For Roslyn we setup FabricBot to request changes on any Arcade update that changes the SDK version (See example https://github.com/dotnet/roslyn/pull/55436#pullrequestreview-723279426). This gives us a chance to downgrade the SDK and take the Arcade changes if they are passing.

**Comment by vzarytovskii** [comment]
Yeah, we have the auto PR from arcade, but for some reason it tries to update to a non-released bits, which breaks our normal use case for VS testing/rebuild.
I've downgraded to latest preview bits.

**Comment by vzarytovskii** [comment]
Fixed by #11988, we are using public builds in CI, also gotta check on every arcade PR if it's installable from public sources.
